callbacks.js

```diff
@@ -1,38 +1,37 @@
-function isVowel (value) {
-    vowels = 'aeiouAEIOU'
-    // if the value is a single letter string and a vowel
-    if(value.length == 1 && vowels.includes(value) ){
-        return true
-    } else {
-        return false
+    function isVowel (value) {
+        vowels = 'aeiouAEIOU'
+        // if the value is a single letter string and a vowel
+        if(value.length == 1 && vowels.includes(value) ){
+            return true
+        } else {
+            return false
```

Comment:
You don't need the else here.

This can be something like this
```js
function isVowel(value) {
  const VOWELS = ['a', 'e' 'i', 'o', 'u'];
  return value.length === 1 && VOWELS.includes(value);
}
```
the result of the return value will always be a boolean which is exactly what you want :)